README.md

```diff
@@ -75,6 +75,21 @@ yarn lint
 npm run lint
 ```
 
+## Unit testing
+The following command allow you to run test with jest.
+
+**for `yarn` package manager**
+
+```
+yarn test
```

Comment:
i checked the package.json and the command for unit test was `yarn test:unit` or `npm run test:unit`